### ISSUE TITLE
ERR_PROXY_CONNECTION_FAILED (-130)

### ISSUE THREAD

**Issue by mavidot** [mattermost/desktop#934]
Hi,
Since 3 days, i'm not able to connect to my society Mattermost server with my Mattermost Desktop 4.2.0 (same with 4.1.2 version). 
Before, it was working like a charm.
I can't find the reason why the problem suddently occured. 
The message on this white page is : 

> Cannot connect to Mattermost
> 
> We're having trouble connecting to Mattermost. If refreshing this page (Ctrl+R or Command+R) does not work please verify that:
> 
> Your computer is connected to the internet.
> The Mattermost URL https://mattermost.dockr.bluexml.com/mysociety is correct.
> You can reach https://mattermost.dockr.bluexml.com/mysociety from a browser window.
> 
> ERR_PROXY_CONNECTION_FAILED (-130)

When i try to access the server from a browser, all is ok. 

I'm working on Mageia (ex-Mandriva) and I doesn't have any proxy setting. 
Do you have an idea of a configuration which could help me to resolve this issue ? Or how I could see some error logs ? 

Thanks

**Comment by wget** [comment]
@mavidot Hi there. thanks for your report. In order for me to try to reproduce the issue, would you mind answering:
* whether you use Mageia 6.1 or any older version?
* from where did you download and install the Desktop app from?
* if I still have issues reproducing the issue, getting ready to create a test account on your instance.
* are you running the EE edition of Mattermost?

**Comment by mavidot** [comment]
Ok, I have resolved the issue this weekend. It was not about mattermost. It was the "Network Manager" tool of Mageia which was broken and did not want to start. It was neither stopped or started. It was on an intermediate state. And I don't know why a proxy configuration was set inside of it (I have never done this). 
But now, the Mageia Network Manager is OK, I have disabled the proxy configuration and all is OK. 
Thank's for attention. 
Have a nice day because Mattermost is healthy :-)
